test-utils/test-runner/src/utils.rs

```diff
@@ -150,3 +124,15 @@ pub fn default_config(task_executor: TaskExecutor, mut chain_spec: Box<dyn Chain
 		transaction_storage: TransactionStorageMode::BlockBody,
 	}
 }
+
+/// Produce a task executor given a handle to a tokio runtime
+pub fn task_executor(handle: Handle) -> TaskExecutor {
+	let task_executor = move |fut, task_type| match task_type {
+		TaskType::Async => handle.spawn(fut).map(drop),
+		TaskType::Blocking => handle
+			.spawn_blocking(move || futures::executor::block_on(fut))
```

Comment:
is that a common thing to run one executor inside of the other? seems weird to me

Reply:
yeah it's literally how sc_cli does it

https://github.com/paritytech/substrate/blob/d3db3c1d1011983ba7565e342910a6d2095625c8/client/cli/src/runner.rs#L129-L136